[Docs] Add docs for Qwen3-VL image and video support

adarshxs opened the pull request:
<!-- Thank you for your contribution! Please follow these guidelines to enhance your pull request. If anything is unclear, submit your PR and reach out to maintainers for assistance. Join our Slack community at https://slack.sglang.ai to discuss further. -->

Added standalone page for Qwen3 VL model family.

cc @mickqian 
<!-- Describe the purpose and goals of this pull request. -->

Commit:
add qwen3 vl docs

Commit:
add to index

adarshxs committed:
Update qwen3_vl.md

merrymercy reviewed (changes requested):
1. list usage for both fp8 and non fp8
2. list the command for more hardware (A100, H200)
3. try to match the vllm repice https://docs.vllm.ai/projects/recipes/en/latest/Qwen/Qwen3-VL.html

adarshxs committed:
Revise Qwen3-VL launch commands and recommendations

Updated launch commands and added hardware-specific recommendations for Qwen3-VL model in SGLang documentation.

adarshxs committed:
lint

adarshxs committed:
Merge branch 'main' into qwen3-vl-docs
